nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/nuxeo.defaults

```diff
@@ -15,6 +15,10 @@ nuxeo.db.embeddedDatasources=true
 nuxeo.db.xaMode=false
 nuxeo.db.commonds=jdbc/nuxeo
 
+nuxeo.db.blocking-timeout-millis=10000
+nuxeo.db.idle-timeout-minutes=0
+nuxeo.db.min-pool-size=5
+
```

Comment:
I think this is done to follow the new pattern in master where there are no more templates-tomcat